Project/CMake/CMakeLists.txt

```diff
@@ -6,7 +6,9 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
 set(MediaInfoLib_MAJOR_VERSION 17)
 set(MediaInfoLib_MINOR_VERSION 10)
-set(MediaInfoLib_VERSION ${MediaInfoLib_MAJOR_VERSION}.${MediaInfoLib_MINOR_VERSION})
+set(MediaInfoLib_PATCH_VERSION 0)
+set(MediaInfoLib_VERSION ${MediaInfoLib_MAJOR_VERSION}.${MediaInfoLib_MINOR_VERSION}.${MediaInfoLib_PATCH_VERSION})

```

Comment:
Do we need the PATCH version here? I would add patch version in displayed number only for patch not 0.